### PR TITLE
Adding support for query offset and TTLs

### PR DESCRIPTION
**Pull request by brimoor**

### CODE REVIEW

**Comment by brimoor** [comment]
@hoddr please test this in dev or staging with the latest API deployed before merging.

File: lib/api.js [hunk 1]
```diff
@@ -231,14 +242,15 @@ class API {
    * @async
    * @param {JobsQuery} jobsQuery A JobsQuery instance defining the customized
    *   jobs query to perform
-   * @returns {Array} an array of objects describing the query results
+   * @returns {Array} an object containing the query results and total number
```
Comment (hoddr) [inline lib/api.js]: Shouldn't the type be Object for these then?

**Review by hoddr — APPROVED** [review]
LGTM (tested offset and limits w/ new query changes) minus comment in doc types

File: lib/api.js (hunk 1 above)
Comment (brimoor) [reply lib/api.js]: good catch; thanks

**Pull request merged**